index.html

```diff
@@ -10610,8 +10610,8 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</p>
 				<p>User agents MUST NOT expose the <code>aria-brailleroledescription</code> property if any of the following conditions exist:</p>
 				<ol>
-					<li>The element to which <code>aria-brailleroledescription</code> is applied does not have a valid WAI-ARIA <code>role</code> or does not have an implicit WAI-ARIA role semantic.</li>
-					<li>The value of <code>aria-brailleroledescription</code> is empty or contains only whitespace characters or contains only Braille Pattern dots-0 (U+2800).</li>
+					<li>The value of <code>aria-brailleroledescription</code> is empty or contains only whitespace characters or the empty Braille pattern: dots-0 (U+2800).</li>
```

Comment:
The old version feels slightly cleaner to me - I could imagine someone misreading the new phrasing to say UAs must ignore any value that contains an occurrence of dots-0 (as opposed to consisting only of dots-0). But I'm happy to yield to native speakers since I doubt implementors will misread it.

Reply:
On second thought, I misread it (sic). The change would now expand the cases to a mix of whitspace and dots-0. That sounds good to me.